tools/chpl-language-server/test/diagnostics.py

```diff
@@ -53,11 +53,22 @@ async def test_incorrect_cross_file_diagnostics(client: LanguageClient):
     async with source_files(client, A=fileA, B=fileB) as docs:
         await save_file(client, docs("B"), docs("A"))
         assert len(client.diagnostics[docs("A").uri]) == 0
-        assert len(client.diagnostics[docs("B").uri]) == 1
-        assert (
-            "an implicit module named 'B' is being introduced to contain file-scope code"
-            in client.diagnostics[docs("B").uri][0].message
-        )
+        assert len(client.diagnostics[docs("B").uri]) >= 1
```

Comment:
recommend setting this to `== 2` to catch further added error messages if they occur.